scripts/tt_boot_fs.py

```diff
@@ -290,6 +290,7 @@ def loads(tag: str, data: dict[str, Any], alignment: FileAlignment, env: dict):
                 raise ValueError(
                     f"{tag} padto value {padto} is < the binary size {len(binary)}"
                 )
+            binary += bytes(padto - len(binary))
```

Comment:
I think we may also want to check if padto is a multiple of the `alignment` argument, and then default `padto` using the `_align_up()` function.